src/worknet/Commands/ContractCommand.Storage.Get.cs

```diff
@@ -0,0 +1,60 @@
+using System.ComponentModel.DataAnnotations;
+using System.IO.Abstractions;
+using McMaster.Extensions.CommandLineUtils;
+using Neo;
+using Neo.BlockchainToolkit.Models;
+using NeoWorkNet.Node;
+
+namespace NeoWorkNet.Commands
+{
+  partial class ContractCommand
+  {
+    // [Command("storage", Description = "Manage neo-worknet values")]
+    // [Subcommand(typeof(Update))]
+    internal partial class Storage
+    {
+      [Command("get", Description = "Update a value in neo-worknet")]
+      internal class Get
+      {
+        readonly IFileSystem fs;
+        public Get(IFileSystem fileSystem)
+        {
+          this.fs = fileSystem;
+        }
+
+        [Argument(0, Description = "Contract name or hash")]
+        [Required]
+        internal string Contract { get; init; } = string.Empty;
+
+        [Argument(1, Description = "Key")]
+        [Required]
+        internal string Key { get; init; } = string.Empty;
+        internal async Task<int> OnExecuteAsync(CommandLineApplication app, IConsole console)
+        {
+          try
+          {
+            var (chain, filename) = await fs.LoadWorknetAsync(app).ConfigureAwait(false);
+            var node = new WorkNetNode(chain, filename);
+            ContractInfo? contractInfo = ContractCommand.FindContractInfo(chain, Contract);
+            var writer = console.Out;
+            byte[]? value = node.GetStorage(contractInfo, Convert.FromHexString(Key.Substring(2)));
+            var stringValue = value == null ? string.Empty : $"0x{Convert.ToHexString(value)}";
```

Comment:
If there is no value for the specified key, command should say something like `key not found`